time_tracker_api/users/users_namespace.py

```diff
@@ -25,17 +25,30 @@
             description='Email of the user that belongs to the tenant',
             example=faker.email(),
         ),
-        'role': fields.String(
+        'role': NullableString(
             title="User's Role",
             max_length=50,
             description='Role assigned to the user by the tenant',
-            example=faker.word(['admin']),
+            example=faker.word(['time-tracker-admin']),
         ),
     },
 )
 
 user_response_fields.update(common_fields)
 
+user_input_fields = ns.model(
```

Comment:
Should not this be named something like `role_input_fields` instead of `user_input_fields `??
I see you are only sending the role attribute

Reply:
Great, I think user_role_input_fields would be better, because is only about user_role.